### DISCUSSION TITLE
什么时候需要用到Helm部署？只有k8s集群能运行graphscope吗？

### DISCUSSION THREAD

**Discussion by ghost** [alibaba/GraphScope#444] [Q&A] [answered]
.

**Comment by lidongze0629** [comment, marked as answer]
Hi, @illcat. Welcome to the GraphScope discussions.

**When I need to deploy GraphScope with Helm?**

As a package manager for Kubernetes, [Helm Deployment](https://artifacthub.io/packages/helm/graphscope/graphscope) is better to deploy GraphScope on Kubernetes easily. What's more, only one session can be connected to the GraphScope service at the same time,  but with helm deployment, you can reconnect the same service after the previous session closed.

**Is Kubernetes an essential to run GraphScope?**

Following GraphScope [FAQ](https://graphscope.io/docs/frequently_asked_questions.html)
> GraphScope supports build and run on local in a single machine, such as ubuntu20. However, GraphScope depends on many third-party libraries and projects. To make our life easier, we suggest you build and run GraphScope on k8s with our provided dev-image and release-image.

If you still want to run GraphScope locally, now, you can follow [this](https://github.com/alibaba/GraphScope/blob/main/.github/workflows/local.yml) to build and run GraphScope on ubuntu20.
